client/src/components/maps/MyMap.js

```diff
@@ -1,33 +1,52 @@
 import React from "react";
 import "../../styles/App.css";
 //import { AddPin } from '../AddPin.js'
-import { Form } from '../Form.js'
-import { GoogleMap, useLoadScript, Marker, InfoWindow } from "@react-google-maps/api"
+import { Form } from "../Form.js";
+import {
```

Comment:
could we improve this formatting?

Reply:
My formatter does it automatically because the line is too long. But once its merged, someone could change it back to a one-liner on their side.